src/core/matrix.c

```diff
@@ -125,19 +168,88 @@ igraph_error_t igraph_matrix_complex_realimag(const igraph_matrix_complex_t *v,
     return IGRAPH_SUCCESS;
 }
 
+/**
+ * \ingroup matrix
+ * \function igraph_matrix_complex_create
+ * \brief Creates a complex matrix from a real and imaginary part.
+ *
+ * \param v Pointer to an uninitialized complex matrix.
+ * \param real Pointer to the real part of the complex matrix.
+ * \param imag Pointer to the imaginary part of the complex matrix.
+ * \return Error code.
+ *
+ * Time complexity: O(n),
+ * n is the
+ * number of elements in the matrix.
+ */
+
 igraph_error_t igraph_matrix_complex_create(igraph_matrix_complex_t *v,
```

Comment:
Let's call the matrix m instead of v, both in the argument list and in the docs. This notation is probably the result of a copy-paste form vector functions.